### PR TITLE
[DOC-5595][DOC-5343][DOC-5250][DOC-3474] v22.2 misc deployment & operations updates

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-5595] Update Go version in output
[DOC-5343] Changes to --max-offset can be applied using a rolling upgrade
[DOC-5250] Add new range.snapshots metrics to references
[DOC-3474] System files except for a defined list are now included in debug.zip

## Previews
[v22.2/ui-custom-chart-debug-page.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/ui-custom-chart-debug-page.html)
[v22.2/when-to-use-regional-vs-global-tables.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/when-to-use-regional-vs-global-tables.html)
[v22.2/regional-tables.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/regional-tables.html)
[v22.2/global-tables.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/global-tables.html)
[v22.2/multiregion-overview.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/multiregion-overview.html)
[v22.2/choosing-a-multi-region-configuration.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/choosing-a-multi-region-configuration.html)
[v22.2/recommended-production-settings.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/recommended-production-settings.html)
[v22.2/operational-faqs.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/operational-faqs.html)
[v22.2/cockroach-debug-zip.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/cockroach-debug-zip.html)
[v22.2/cockroach-start.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/cockroach-start.html)
[v22.2/node-shutdown.md](https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/node-shutdown.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

File: _includes/v22.2/metric-names-serverless.md [hunk 1]
```diff
@@ -136,6 +136,13 @@ Name | Description
 `range.raftleadertransfers` | Number of raft leader transfers
 `range.removes` | Number of range removals
 `range.snapshots.generated` | Number of generated snapshots
+`range.snapshots.recv-in-progress` | Number of non-empty snapshots in progress on a receiver store
+`range.snapshots.recv-queue` | Number of queued non-empty snapshots on a receiver store
+`range.snapshots.recv-total-in-progress` | Number of empty and non-empty snapshots in progress on a receiver store
+`range.snapshots.send-in-progress` | Number of non-empty snapshots in progress on a sender store
+`range.snapshots.send-queue` | Number of queued non-empty snapshots on a sender store
+`range.snapshots.send-total-in-progress` | Number of empty and non-empty in-progress
+snapshots on a sender store
```
Comment (taroface) [inline _includes/v22.2/metric-names-serverless.md:145]: Looks like an extra line break above this.

File: _includes/v22.2/metric-names-serverless.md [hunk 2]
```diff
@@ -148,10 +155,10 @@ Name | Description
 `replicas.quiescent` | Number of quiesced replicas
 `replicas.reserved` | Number of replicas reserved for snapshots
 `replicas` | Number of replicas
-`requests.backpressure.split` | Number of backpressured writes waiting on a Range split. A Range will backpressure (roughly) non-system traffic when the range is above the configured size until the range splits. When the rate of this metric is nonzero over extended periods of time, it should be investigated why splits are not occurring. 
+`requests.backpressure.split` | Number of backpressured writes waiting on a Range split. A Range will backpressure (roughly) non-system traffic when the range is above the configured size until the range splits. When the rate of this metric is nonzero over extended periods of time, it should be investigated why splits are not occurring.
```
Comment (taroface) [inline _includes/v22.2/metric-names-serverless.md:158]:
```suggestion
`requests.backpressure.split` | Number of backpressured writes waiting on a range split. A range will backpressure (roughly) non-system traffic when the range is above the configured size until the range splits. When the rate of this metric is nonzero over extended periods of time, it should be investigated why splits are not occurring.
```

File: _includes/v22.2/metric-names-serverless.md [hunk 3]
```diff
@@ -148,10 +155,10 @@ Name | Description
...
-`requests.slow.raft` | Number of requests that have been stuck for a long time in the replication layer. An (evaluated) request has to pass through the replication layer, notably the quota pool and raft. If it fails to do so within a highly permissive duration, the gauge is incremented (and decremented again once the request is either applied or returns an error). A nonzero value indicates range or replica unavailability, and should be investigated. 
+`requests.slow.lease` | Number of requests that have been stuck for a long time acquiring a lease. This gauge registering a nonzero value usually indicates range or replica unavailability, and should be investigated. In the common case, we also expect to see 'requests.slow.raft' to register a nonzero value, indicating that the lease requests are not getting a timely response from the replication layer.
```
Comment (taroface) [inline _includes/v22.2/metric-names-serverless.md:160]: Elsewhere the metrics names don't have enclosed punctuation, so suggesting this for consistency

```suggestion
`requests.slow.lease` | Number of requests that have been stuck for a long time acquiring a lease. This gauge registering a nonzero value usually indicates range or replica unavailability, and should be investigated. In the common case, we also expect to see requests.slow.raft to register a nonzero value, indicating that the lease requests are not getting a timely response from the replication layer.
```

**Review by taroface — APPROVED** [review]
LGTM. Couple nits (that I think should appear this time).

File: _includes/v22.2/metric-names-serverless.md (hunk 1 above)
Comment (mdlinville) [reply _includes/v22.2/metric-names-serverless.md:145]: I'm not sure what you mean. I don't see any extra breaks in the source (based on looking at the line numbers) or in https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/ui-custom-chart-debug-page.html.

Comment (taroface) [reply _includes/v22.2/metric-names-serverless.md:145]: I'm referring to these lines: 
image

File: v22.2/cockroach-start.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ Flag | Description
...
+<a name="flags-max-offset"></a>`--max-offset` | The maximum allowed clock offset for the cluster. If observed clock offsets exceed this limit, servers will crash to minimize the likelihood of reading inconsistent data. Increasing this value will increase the time to recovery of failures as well as the frequency of uncertainty-based read restarts.<br><br>Nodes can run with different values for `--max-offset`, but only for the purpose of updating the setting across the cluster using a rolling upgrade.<br><br>**Default:** `500ms`
```
Comment (mdlinville) [inline v22.2/cockroach-start.md:67]: @nickvigilante Is the check on line 57 new? I believe it's in conflict with https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#capitalization-rules.

File: _includes/v22.2/metric-names.md [hunk 5]
```diff
@@ -133,6 +133,12 @@ Name | Help
 `range.adds` | Number of range additions
 `range.raftleadertransfers` | Number of Raft leader transfers
 `range.removes` | Number of range removals
+`range.snapshots.recv-in-progress` | Number of non-empty snapshots in progress on a receiver store
+`range.snapshots.recv-queue` | Number of queued non-empty snapshots on a receiver store
+`range.snapshots.recv-total-in-progress` | Number of empty and non-empty snapshots in progress on a receiver store
+`range.snapshots.send-in-progress` | Number of non-empty snapshots in progress on a sender store
+`range.snapshots.send-queue` | Number of queued non-empty snapshots on a sender store
+`range.snapshots.send-total-in-progress` | Number of empty and non-empty in-progress snapshots on a sender store
```
Comment (mdlinville) [inline _includes/v22.2/metric-names.md:141]: @nickvigilante I think the Linter for "hit" on line 177 is a false positive and we should have an exception for "cache hit".

File: _includes/v22.2/metric-names-serverless.md [hunk 6]
```diff
@@ -130,28 +130,35 @@ Name | Description
...
-`range.raftleadertransfers` | Number of raft leader transfers
+`range.raftleadertransfers` | Number of Raft leader transfers
 `range.removes` | Number of range removals
 `range.snapshots.generated` | Number of generated snapshots
+`range.snapshots.recv-in-progress` | Number of non-empty snapshots in progress on a receiver store
+`range.snapshots.recv-queue` | Number of queued non-empty snapshots on a receiver store
+`range.snapshots.recv-total-in-progress` | Number of empty and non-empty snapshots in progress on a receiver store
+`range.snapshots.send-in-progress` | Number of non-empty snapshots in progress on a sender store
+`range.snapshots.send-queue` | Number of queued non-empty snapshots on a sender store
+`range.snapshots.send-total-in-progress` | Number of empty and non-empty in-progress
```
Comment (mdlinville) [inline _includes/v22.2/metric-names-serverless.md:144]:
```suggestion
`range.snapshots.send-total-in-progress` | Number of empty and non-empty in-progress on a sender store
```

File: _includes/v22.2/metric-names-serverless.md [hunk 7]
```diff
@@ -130,28 +130,35 @@ Name | Description
...
+snapshots on a sender store
```
Comment (mdlinville) [inline _includes/v22.2/metric-names-serverless.md:145]:
```suggestion
```

**Pull request merged**